src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -125,6 +127,8 @@ const ActiveState = ({
     shouldUseUAA,
     translations,
 }: Props): React.Node => {
+    const [editingCommentIds, setEditingCommentIds] = React.useState<string[]>([]);
```

Comment:
I actually did do this at first but we ultimately decided against doing that since it would be a one of case of us doing that. No where else do they do that. Performance-wise, using context is the best move but for readability reasons it would be out of place.